### PR TITLE
odim-5334: Update ODIM to latest DMTF schema 2021.2

### PR DESCRIPTION
**Pull request by hemkapoor**
Update ODIM to latest DMTF schema 2021.2

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: lib-utilities/etc/Base.1.10.0.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "@Redfish.Copyright": "Copyright 2014-2021 DMTF. All rights reserved.",
     "@Redfish.License": "Creative Commons Attribution 4.0 License.  For full text see link: https://creativecommons.org/licenses/by/4.0/",
-    "@odata.type": "#MessageRegistry.v1_4_0.MessageRegistry",
+    "@odata.type": "#MessageRegistry.v1_4_2.MessageRegistry",
```
Comment (amar-shalgar) [inline lib-utilities/etc/Base.1.10.0.json]: Please donot change anything in the json files. these files should come from redfish spec. update the json to latest. applies for all the json's

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address comments

File: lib-utilities/etc/Base.1.10.0.json (hunk 1 above)
Comment (hemkapoor) [reply lib-utilities/etc/Base.1.10.0.json]: Made the required changes

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Pull request merged**